docs/release/support_matrix_v2_0.md

```diff
@@ -1,73 +1,273 @@
+Under such a support agreement, Intel partners that offer DAOS
+Commercial Support to their end customers will provide the DAOS
+Level-1 and Level-2 support. They can then escalate Level-2 support
+tickets to the Intel Level-3 support team
+through a dedicated JIRA path with well-defined SLAs.
+Please refer to the
+[intel.com landing page for DAOS](https://www.intel.com/content/www/us/en/high-performance-computing/daos.html)
+for information on the DAOS partner ecosystem.
+
+This document describes the supported environments for Intel Level-3 support.
+Partner support offerings may impose further constraints, for example if they
+include DAOS support as part of a more general cluster support offering
+with its own release cycle.
+
+Some members of the DAOS community have reported successful compilation
+and basic testing of DAOS in other environments (for example on ARM64
+platforms, or on other Linux distributions). Those activities are highly
+appreciated community contributions. However such environments are
+not currently supported in a production environment.
+
+
+## Hardware platforms supported for DAOS Servers
+
+DAOS Version 2.0 supports the x86\_64 architecture.
 
 DAOS servers require Storage Class Memory (SCM) that is supported by
 [PMDK](https://github.com/pmem/pmdk).
 (SCM can be emulated by DRAM for development and testing purposes,
 but this is not supported in a production environment.)
-DAOS has been validated with
+DAOS Version 2.0 has been validated with
 [Intel Optane Persistent Memory 100 Series](https://ark.intel.com/content/www/us/en/ark/products/series/190349/intel-optane-persistent-memory-100-series.html)
-on 2nd gen Intel Xeon Scalable processors, and
+on 2nd gen Intel Xeon Scalable processors, and with
 [Intel Optane Persistent Memory 200 Series](https://ark.intel.com/content/www/us/en/ark/products/series/203877/intel-optane-persistent-memory-200-series.html)
 on 3rd gen Intel Xeon Scalable processors.
+It is strongly recommended that all memory channels of a DAOS server
+are populated with one DRAM module and one Optane PMem module.
+All Optane PMem modules in a DAOS server must have the same capacity.
+
+While not strictly required, DAOS servers typically include NVMe disks
+for bulk storage, which must be supported by [SPDK](https://spdk.io/).
+(NVMe storage can be emulated by files on non-NVMe storage for development
+and testing purposes, but this is not supported in a production environment.)
+All NVMe disks managed by a single DAOS engine should have identical capacity,
+and it is strongly recommended to use identical drive models.
+
+Note: It is possible and supported that different DAOS engines in a DAOS system
+have different PMem and NVMe configurations, as long as the storage configuration
+_within_ each DAOS engine is homogeneous. The different capabilities of the
+DAOS engines in such an inhomogeneous DAOS system need to be carefully considered
+when creating DAOS pools.
```

Comment:
Any other documentation the the tradeoffs we could link to?

Reply:
based on yesterday's conversations, we'll further restrict this, and will also require the same number of targets on all engines.